hikari/__init__.py

```diff
@@ -72,11 +74,11 @@
 from hikari.files import Rawish
 from hikari.files import Resourceish
 from hikari.guilds import *
-from hikari.impl.bot import BotApp
-from hikari.impl.bot import GatewayBot
-from hikari.impl.rest import ClientCredentialsStrategy
-from hikari.impl.rest import RESTApp
-from hikari.impl.rest_bot import RESTBot
+from hikari.impl import BotApp
+from hikari.impl import ClientCredentialsStrategy
+from hikari.impl import GatewayBot
+from hikari.impl import RESTApp
+from hikari.impl import RESTBot
```

Comment:
This import method is not something we have used until now. We always have the full qualified path to to object (how it was before)